app/classes/Stores/Project.php

```diff
@@ -375,17 +372,50 @@ public function getTemplate($store, $channel)
     }
 
     /**
-     * Get the lang file name for a Store and Channel
+     * Get the lang file name(s) for a Store and Channel. Can be filtered by Section.
      * @param  string $store   Name of the Store
      * @param  string $channel Name of the Channel
-     * @return mixed  String containing the langfile name or False
+     * @param  string $section Name of the Section. Returns all sections by default.
+     * @return mixed  String containing the langfile name(s) or False
      */
-    public function getLangFile($store, $channel)
+    public function getLangFile($store, $channel, $section = '')
     {
         if (! isset($this->templates[$store][$channel]['langfile'])) {
             return false;
         }
 
-        return $this->templates[$store][$channel]['langfile'];
+        $files = $this->templates[$store][$channel]['langfile'];
+
+        switch ($section) {
```

Comment:
I think this part is very complicated, I'll study it tomorrow, but I notice that the Done api returns an empty array while the view for the release channel for Android shows many locales as done, so I think something is wrong in the logic.